R/btw_this.R

```diff
@@ -229,7 +229,8 @@ btw_user_prompt <- function(x) {
   structure(x, class = c("btw_user_prompt", "btw_ignore"))
 }
 
-#' @export
+# https://github.com/RConsortium/S7/issues/501#issuecomment-2494609728
+#' @rawNamespace S3method(base::print, btw_ignore)
```

Comment:
Just a note to future selves, we need this because we're defining a print method for the `BTW` class, and we'll have to use this for any other S3 print methods, if we define any.